ec2utils/ec2uploadimg/ec2uploadimg

```diff
@@ -549,3 +564,6 @@ except EC2UploadImgException as e:
 except Exception as e:
     print(format(e), file=sys.stderr)
     sys.exit(1)
+finally:
+    if setup:
+        setup.clean_up()
```

Comment:
clean up should be unconditional, we create the object unconditionally. Also cleanup needs to be in the region loop or we will leave traces behind in each region but the last. clean_up call also needs to be in the exception handlers.